src/MetricExporter.ts

```diff
@@ -96,135 +87,73 @@ export class DynatraceMetricExporter implements MetricExporter {
 		};
 	}
 
-	export(
-		metrics: MetricRecord[],
-		resultCallback: (result: ExportResult) => void
-	): void {
-		if (metrics.length === 0) {
-			process.nextTick(resultCallback, { code: ExportResultCode.SUCCESS });
-			return;
+	selectAggregationTemporality(instrumentType: InstrumentType): AggregationTemporality {
+		switch (instrumentType) {
+			case InstrumentType.COUNTER:
+			case InstrumentType.HISTOGRAM:
+			case InstrumentType.OBSERVABLE_COUNTER:
+				return AggregationTemporality.DELTA;
+			default:
+				return AggregationTemporality.CUMULATIVE;
```

Comment:
```suggestion
			case InstrumentType.OBSERVABLE_GAUGE:
			case InstrumentType.OBSERVABLE_UP_DOWN_COUNTER:
			case InstrumentType.UP_DOWN_COUNTER:
				return AggregationTemporality.CUMULATIVE;
			default:
				return AggregationTemporality.DELTA;
```
I would flip this around, this way we would automatically also request Deltas when exponential histograms come around without having to change the code. Also, this might be an implementation detail, but Gauges do not have a temporality in OTLP, right? Is there a difference in the aggregation?

Reply:
I'll flip it. There is no real difference I just wanted to make sure all cases were covered.

Reply:
No difference, but futureproofing. Thanks!